openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,38 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
 This specification also defines the following terms. In the case where a term has a definition that differs, the definition below is authoritative for this specification.
 
-Credential:
-:  A set of one or more claims about a subject made by a Credential Issuer. Note that this definition of a term "credential" in this specification is different from that in [@!OpenID.Core] and [@!RFC6749].
+Credential Dataset:
+:  A collection of one or more assertions about a subject, provided by a Credential Issuer. These assertions are also known as claims.
```

Comment:
Use "claims" - not "assertions".  Claims is used pervasively in OpenID and IETF specs.  Assertions is SAML terminology.  We should not reintroduce it.